ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -39,6 +39,7 @@ final case class Config[Extra](
     metricsReportingInterval: Duration,
     trackerRetentionPeriod: FiniteDuration,
     engineMode: EngineMode,
+    enableExperimentalSchemaForIndex: Boolean,
```

Comment:
Is there a reason for calling this 'experimental schema' instead of 'append-only schema'? If we only intend to have additional 'experimental' schema changes on top of the append-only schema then let's call this `enableAppendOnlySchema` or similar, so that it is not confusing (i.e., 'experimental' and 'append-only' refers to the same concept in the code).

Reply:
I concur.

Reply:
I agree as well, I would suggest calling this appendOnly (e.g. enableAppendOnlySchema here), and renaming accordingly above as well if needed